app/services/ping_thing_validation_service.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class PingThingValidationService
```

Comment:
I think that this service main goal is not validation so I'd give it a different name